hypershift-agent-template/templates/hostedcluster.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: hypershift.openshift.io/v1beta1
+kind: HostedCluster
+metadata:
+  name: {{ .Release.Namespace }}-{{ .Release.Name }}
+spec:
+  release:
+    image: quay.io/openshift-release-dev/ocp-release:{{ .Values.ocpVersion }}-x86_64
+  pullSecret:
+    name: pullsecret-cluster
+  sshKey:
+    name: sshkey-cluster
+  networking:
+    clusterNetwork:
+      - cidr: 10.132.0.0/14
+    machineNetwork:
+      - cidr: 192.168.122.0/24
+    networkType: OVNKubernetes
+    serviceNetwork:
+      - cidr: 172.31.0.0/16
+  platform:
+    type: Agent
+    agent:
+      agentNamespace: {{ .Values.agentNamespace }}
+  infraID: {{ .Release.Name }}
+  dns:
+    baseDomain: {{ .Values.baseDnsDomain }}
+  services:
+  - service: APIServer
+    servicePublishingStrategy:
+      type: LoadBalancer
```

Comment:
So let remove the nodeport support?

Reply:
I guess this is just an oversight. It should be

```yaml
- service: APIServer
   servicePublishingStrategy:
     type: {{ .Values.APIPublishingStrategy }}
     {{- if .Values.NodePortAddress  }}
     nodePort:
       address: {{ .Values.NodePortAddress }}
     {{- end }}
```

Reply:
indeed, fixed